pihole

```diff
@@ -62,6 +62,12 @@ function setupLCDFunction {
     exit 1
 }
 
+function queryFunc {
+    domain=$2
+    for list in /etc/pihole/list.*;do echo $list;grep ${domain} $list;done
+    exit 1
```

Comment:
I'm guessing the convention was started by copying the return val for the `sudo` check, where it's properly used, and just continued on from there when the code was refactored. I don't remember offhand if we trap the return values or make any use of them directly. But yes, I do agree that it's not standard convention.